packages/trpc/server/routers/viewer/organizations/_router.tsx

```diff
@@ -114,7 +109,7 @@ export const viewerOrganizationsRouter = router({
     const handler = await importHandler(namespaced("getOtherTeam"), () => import("./getOtherTeam.handler"));
     return handler(opts);
   }),
-  listOtherTeams: authedOrgAdminProcedure.query(async (opts) => {
+  listOtherTeams: authedProcedure.query(async (opts) => {
```

Comment:
It was unnecessarily causing 403 for all non admin members. Instead, let the route itself handle it and return empty array for non admin members,